packages/esm-patient-programs-app/src/programs/programs.resource.tsx

```diff
@@ -17,7 +18,7 @@ export function useEnrollments(patientUuid: string) {
       : null;
 
   return {
-    data: data ? uniqBy(formattedEnrollments, (program) => program.program.uuid) : null,
+    data: data ? uniqBy(formattedEnrollments, (program) => program?.program?.uuid) : null,
```

Comment:
Was causing the `ProgramsForm` tests to fail because it `couldn't find property uuid of undefined`.